tests/usernames.txt

```diff
@@ -4,5 +4,7 @@ bbixler
 cpappenheimer
 GuillermoFidalgo
 henryiii
+prakamya
+roy-cruz
 mthakorecern
-roy-cruz
\ No newline at end of file
+roy-cruz
```

Comment:
You left in the old "roy-cruz" when you moved it. I can fix it here, but something to watch for.

Reply:
Also, this isn't alphabetical.